lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -4562,6 +4580,17 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
                      ValueWithCleanup(adjAccess, makeCleanupFromChildren({})));
   }
 
+#define PROPAGATE_CLEANUP(INST) \
```

Comment:
How about
```suggestion
#define PROPAGATE_BUFFER_CLEANUP(INST) \
```

Reply:
Done in https://github.com/apple/swift/pull/22973/commits/e42706e5b266d61589a06dcb3ee67cfc1a2ac407.